File: CustomView/app/src/main/java/com/android/customview/skin/MainActivity.java
package com.android.customview.skin;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.android.customview.R;

import skin.support.SkinCompatManager;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        // 指定皮肤插件
        SkinCompatManager.getInstance().loadSkin("night.skin", null, SkinCompatManager.SKIN_LOADER_STRATEGY_ASSETS);
    }

//
//    @NonNull
//    @Override
//    public AppCompatDelegate getDelegate() {
//        return SkinAppCompatDelegateImpl.get(this, this);
//    }
}

File: motionlayout2/src/main/java/com/wkk/motionlayoutdemo/MainActivity.kt
package com.wkk.motionlayoutdemo

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.wkk.motionlayoutdemo.databinding.ActivityMainBinding
import com.wkk.motionlayoutdemo.ext.startActivity

/**
 *博客地址：https://mp.weixin.qq.com/s/dF32SX61qTzV-hTC2Rx8EA
 */
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.btnHWTel.setOnClickListener {
            startActivity<HuaweiTelActivity>()
        }

        binding.btnCodeOperation.setOnClickListener {
            startActivity<CodeControlActivity>()
        }

        binding.btnViewPager.setOnClickListener {
            startActivity<ViewPagerDemoActivity>()
        }

        binding.btnEasterEggs.setOnClickListener {
            startActivity<EasterEggs11Activity>()
        }
    }
}